namex-solr-api/devops/vaults.gcp.env

```diff
@@ -21,9 +21,10 @@ SOLR_SVC_NAMEX_LEADER_CORE="op://namex/$APP_ENV/solr-api/SOLR_SVC_NAMEX_LEADER_C
 SOLR_SVC_NAMEX_FOLLOWER_CORE="op://namex/$APP_ENV/solr-api/SOLR_SVC_NAMEX_FOLLOWER_CORE"
 SOLR_SVC_NAMEX_LEADER_URL="op://namex/$APP_ENV/solr-api/SOLR_SVC_NAMEX_LEADER_URL"
 SOLR_SVC_NAMEX_FOLLOWER_URL="op://namex/$APP_ENV/solr-api/SOLR_SVC_NAMEX_FOLLOWER_URL"
-VPC_CONNECTOR="op://namex/$APP_ENV/solr-api/VPC_CONNECTOR"
+VPC_CONNECTOR="op://CD/$APP_ENV/base/VPC_CONNECTOR"
```

Comment:
If you set it this way then the api will not be able to connect to solr. You need to set it to the solr vpc

Reply:
I also don't think you need a new branch for something this small. The CD doesn't work on main so it doesn't matter   if it takes a few tries

Reply:
It will trigger on merge and you can also trigger it manually yes